docs/general/introduction.md

```diff
@@ -49,6 +26,6 @@ If you have questions regarding personal matters like the migration of LYXe, Uni
 
 Suppose it's a vulnerability or something that should not be shared with the public, please reach out to official team members within our developer channel on [Discord](https://discord.gg/lukso). You can send a friend request in order to exchange more detailed information. We are also preparing a workflow to handle investigations and bug bounties in a unified format.
 
-## Does LUKSO have grants or bug bounties?
+## Does LUKSO have a grants programme?
```

Comment:
is it necessary to have it twice?

Reply:
No, probably not. I'll remove it here as I don't think a user would find it useful information in getting started, but a developer might. 

I'll leave it FAQ for now, because we have received that question quite a few times.